docs/desktop/tools/firewallgui.md

```diff
@@ -40,70 +40,68 @@ When you open the app, it will ask for your password. It will also ask again bef
+At the bottom of the panel, you will find the "Change Zone" button, which allows you to assign your connection to another zone. In Permanent mode, you can also create your own custom Zones.
 
 ![a screenshot featuring the Active Bindings panel on the left of the window](images/firewallgui-03.png)
 
 ## Managing Zones
 
-In the first tab on the right panel, you will find the Zone menu. Here you can open and close ports, enable or disable services, add trusted IP addresses for incoming traffic (think local networks), enable port forwarding, add rich rules, and more.
+In the first tab on the right panel, you will find the Zone menu. Here, you can open and close ports, turn services on or off, add trusted IP addresses for incoming traffic (think local networks), enable port forwarding, add rich rules, and more.
 
-For most basic desktop users, this is where you will spend most of your time, and the most useful sub-tabs in this panel will be the ones for configuring services and ports.
+For most basic desktop users, this is where you will spend most of your time, and the most valuable sub-tabs in this panel will be the ones for configuring services and ports.
 
 !!! Note
 
-    If you install your apps and services from the repository, some of them (usually the ones designed for Desktop use) will automatically enable the relevant services or open the proper ports. However, if that does not happen, you can follow the steps below to do it all manually.
+    Install your apps and services from the repository. Some of them (usually the ones designed for Desktop use) will automatically enable the relevant services or open the proper ports. However, if that does not happen, you can follow the steps below to do it all manually.
 
 ### Adding a service to a Zone
 
-Services are popular apps and background services that `firewalld` supports by default. You can quickly and easily enable them by scrolling through the list, and clicking the relevant checkbox.
+Services are popular apps and background services that are `firewalld` and supported by default. You can quickly and easily enable them by scrolling through the list and clicking the relevant checkbox.
 
-Now, if you have installed KDE Connect* to help sync your desktop with other devices, and you want to allow it through your firewall so it will actually work, you would:
+Now, if you have installed KDE Connect* to help sync your desktop with other devices, and you want to allow it through your firewall so it will work, you would:
 
-1. First, select the zone you want to edit. Just go with the default public zone, for this example.
+1. First, select the zone you want to edit. Just go with the default public zone for this example.
 2. Scroll down the list, and select "kdeconnect".
 3. If you are in the Runtime configuration mode, do not forget to click "Runtime To Permanent" and "Reload Firewalld" in the options menu.
 
 \* Available in the EPEL repository.
```

Comment:
?

Reply:
It's a reference to an earlier "*"... @SergeCroise. @gannazhyrnova in my editor, it does not like "etc." and says you should use "and so on" rather than "etc." ... I'm leaving this change however.